### PR TITLE
Add default to snmp community api_data

### PR DESCRIPTION
**Pull request by Tr4sK**
##### SUMMARY

Add missing default attribute to api_data

##### ISSUE TYPE

- Feature Pull Request

##### ADDITIONAL INFORMATION

  - snmp community

### CODE REVIEW

File: changelogs/fragments/311-add-defaults-fields-snmp-community.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes
+  - api_modify - add default field to ``snmp community`` (https://github.com/ansible-collections/community.routeros/pull/311).
```
Comment (felixfontein) [inline changelogs/fragments/311-add-defaults-fields-snmp-community.yml:2]: This also affects the api_info module:
```suggestion
  - api_modify, api_info - add default field to ``snmp community`` (https://github.com/ansible-collections/community.routeros/pull/311).
```

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution!

File: changelogs/fragments/311-add-defaults-fields-snmp-community.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+minor_changes
```
Comment (felixfontein) [inline changelogs/fragments/311-add-defaults-fields-snmp-community.yml:1]: This will fix the failing tests:
```suggestion
minor_changes:
```

**Comment by felixfontein** [comment]
Can you please remove the `@` from the commit messages? With the `@` in them I get a notification every single time these commits are pushed somewhere, which is a bit annoying :)

**Comment by Tr4sK** [comment]
Is there a way to not change api_modify ?  
Since, the API can list the default snmp community but do not accept that parameter as a modification.  
Any way to mark it as readonly ?

File: plugins/module_utils/_api_data.py [hunk 3]
```diff
@@ -1086,6 +1086,7 @@ def join_path(path):
             fully_understood=True,
             primary_keys=('name', ),
             fields={
+                'default': KeyInfo(),
```
Comment (felixfontein) [inline plugins/module_utils/_api_data.py:1089]:
```suggestion
                'default': KeyInfo(read_only=True),
```

**Review by felixfontein — COMMENTED** [review]
Ah, so that field is read-only and cannot be set? In that case you can use `read_only=True`:

File: changelogs/fragments/311-add-defaults-fields-snmp-community.yml [hunk 4]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - api_modify, api_info - add ``default`` field to ``snmp community`` (https://github.com/ansible-collections/community.routeros/pull/311).
```
Comment (felixfontein) [inline changelogs/fragments/311-add-defaults-fields-snmp-community.yml:2]: Maybe in that case also add:
```suggestion
  - api_modify, api_info - add read-only ``default`` field to ``snmp community`` (https://github.com/ansible-collections/community.routeros/pull/311).
```

**Comment by felixfontein** [comment]
@Tr4sK thanks for your contribution!

**Pull request merged**